dev-docs/source/ReleaseChecklist.rst

```diff
@@ -68,6 +68,12 @@ Unscripted and Final Testing (technical tasks)
    `open-release-testing <http://builds.mantidproject.org/view/All/job/open-release-testing/>`__
    to create the release branch and prepare build jobs
 *  Check state of all open pull requests for this milestone and decide which should be kept for the release, liase with PM on this. Move any pull requests not targeted for release out of the milestone and run `update-pr-base-branch.py <https://github.com/mantidproject/mantid/blob/master/tools/scripts/update-pr-base-branch.py>`__ to update the base branches of those pull requests.
+*  Inform other developers that release-next has been created by adapting/posting the following announcement:
+
+  .. code
+  
+  The release branch for <version>, called release-next, has now been created: https://github.com/mantidproject/mantid/tree/release-next.  If you've not worked with the release/master-branch workflow before then please take a moment to familiarise yourself with the process: http://developer.mantidproject.org/GitWorkflow.html#code-freeze. The part about ensuring new branches have the correct parent is the most important part (although this can be corrected afterwards). All PRs that were open before the release branch are fine as their history is the same as master.
```

Comment:
Could we reword to clarify the last sentance -> "All PRs that were created before the release branch was created ..." ?

Reply:
I've tweaked the wording of the final sentence and added your suggestions + some more clarification. Does it read better?